src/components/Filters.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { getArticlesByTopic } from '../utils/articlesApi';
+import { Link, useParams } from 'react-router-dom';
+import ArticleList from './ArticleList';
+
+const Filters = ({ articles, setArticles }) => {
+  const { topic } = useParams();
+
+  useEffect(
+    () => {
+      getArticlesByTopic(topic).then((articlesFromApi) => {
+        console.log(articlesFromApi);
+        setArticles(articlesFromApi.articles);
+      });
+    },
+    [ setArticles ]
+  );
+  console.log(topic);
```

Comment:
please remove console logs

Reply:
Forgot to delete the component as a whole. Now removed